File: debug_openapi.py
import json

try:
    with open('/home/baga/Code/admixcenter/openapi.json', 'r') as f:
        data = json.load(f)
        print("Root keys:", data.keys())
        if 'paths' in data:
            print("Number of paths:", len(data['paths']))
            # Print first 10 paths
            for i, path in enumerate(data['paths']):
                if i < 10:
                    print(path)
        else:
            print("'paths' key not found")
except Exception as e:
    print(f"Error: {e}")

File: debug_openapi.php
<?php
try {
    $content = file_get_contents('openapi.json');
    if ($content === false) {
        file_put_contents('debug_output.txt', 'Failed to read file');
    } else {
        $paths = [];
        $data = json_decode($content, true);
        if (json_last_error() !== JSON_ERROR_NONE) {
            file_put_contents('debug_output.txt', 'JSON Decode Error: ' . json_last_error_msg());
        } else {
            if (isset($data['paths'])) {
                foreach (array_keys($data['paths']) as $path) {
                    if (strpos($path, 'system') !== false || strpos($path, 'status') !== false) {
                        $paths[] = $path;
                    }
                }
                file_put_contents('debug_output.txt', implode("\n", $paths));
            } else {
                file_put_contents('debug_output.txt', 'No paths found');
            }
        }
    }
} catch (Exception $e) {
    file_put_contents('debug_output.txt', 'Exception: ' . $e->getMessage());
}
